junit-jupiter-engine/src/test/java/org/junit/jupiter/api/condition/EnabledIfSystemPropertyIntegrationTests.java

```diff
@@ -25,59 +30,84 @@
  */
 class EnabledIfSystemPropertyIntegrationTests {
 
-	private static final String KEY = "EnabledIfSystemPropertyTests.key";
+	private static final String KEY1 = "EnabledIfSystemPropertyTests.key1";
+	private static final String KEY2 = "EnabledIfSystemPropertyTests.key2";
 	private static final String ENIGMA = "EnabledIfSystemPropertyTests.enigma";
```

Comment:
You removed the prefix everywhere but here, don't know if it's intentional?

Reply:
Good catch. That was not intentional. I'll fix it.